config/routes.rb

```diff
@@ -1,3 +1,10 @@
 Rails.application.routes.draw do
   # For details on the DSL available within this file, see https://guides.rubyonrails.org/routing.html
+  get "users/new" => "users#new", :as => :new_user
+  post "users/new" => "users#new"
+
+  get 'users/edit', to: 'users#edit', as: :edit_user
+  patch 'users/:id', to: 'users#update'
```

Comment:
- [ ] You don't need these if you're using `resources :users, only: [:new, :create, :edit, :update]`. Its a duplication